core/storage/feedback/gae_models.py

```diff
@@ -204,16 +299,34 @@ def get_messages(cls, exploration_id, thread_id):
 
     @classmethod
     def get_most_recent_message(cls, exploration_id, thread_id):
+        """Returns the most recent messages from the given thread, ordered
```

Comment:
I was wrong. It returns the last message added in the thread. Should I write "Returns the last message added in the thread" ?